client/app/lib/providers/computecontroller.coffee

```diff
@@ -37,7 +37,7 @@ require './config'
 module.exports = class ComputeController extends KDController
 
 
-  @providers = globals.config.providers._getSupportedProviders()
+  PROVIDERS  = globals.config.providers._getSupportedProviders()
```

Comment:
most of the time where I need supported providers list I also need providers as well so I kept it in the same object for consistency.